sdk/util.js

```diff
@@ -7,8 +7,8 @@ const headers = {
     protocol: AVAILABLE_PROTOCOLS.ETH,
 };
 
-const WS_PROXY_URL = "ws://nft-sales-websocket.dappradar.com";
```

Comment:
ws://nft-sales-websocket.dappradar.com

Reply:
When using this WS it does not return any valid data for Polygon/Matic... we can only receive data when using wss://rpc-mainnet.matic.quiknode.pro